### DISCUSSION TITLE
When are the mods gonna be fixed T^T

### DISCUSSION THREAD

**Discussion by LyomaBeans** [thatskymod/Sky-CotL-Scripts#123] [Q&A] [answered]
dude ts is comin and shattered ts toooooo mannnnn i dont have time to be candle runnin ):

**Comment by thatskymod** [comment, marked as answer]
Greetings @LyomaBeans,

while all modifications in this repository undergo rigorous testing, there's always a small chance that minor issues slip through the cracks.

For your convenience, I'm going to link your previous thread, #116. As it contains more comprehensive details about your situation, I recommend continuing the discussion there. You're likely to receive faster, more personalized assistance in that thread.

Thank you for your contribution.
